### PR TITLE
docs: Remove known DoH issue with Firefox

### PR DESCRIPTION
**Pull request by jamilbk**
This has been a long-standing issue.

The base PR fixes the issue for Firefox, and apparently all other browsers will _not_ change your DNS server, only opportunistically enable DoH if it finds your current servers to support it.

### CODE REVIEW

**Review by ReactorScram — APPROVED** [review]
Wow never thought this one could be fixed, nice

File: website/src/app/kb/client-apps/ios-client/readme.mdx [hunk 1]
```diff
@@ -100,7 +100,5 @@ We will add troubleshooting steps here in the future.
 
 - If another VPN app is running on the system, Firezone will not work.
   [#4733](https://github.com/firezone/firezone/issues/4733)
-- Some browsers (e.g. Safari) break DNS routing
```
Comment (thomaseizinger) [inline website/src/app/kb/client-apps/ios-client/readme.mdx:103]: I haven't had any issues with Safari but I also don't have an iOS device.

**Review by thomaseizinger — APPROVED** [review]
Not sure if we should remove it for all browsers? I didn't have any issues with Chrome or Safari on MacOS in their default configuration so I guess it is fixed? Was it ever broken for them?

**Comment by jamilbk** [comment]
> Not sure if we should remove it for all browsers? I didn't have any issues with Chrome or Safari on MacOS in their default configuration so I guess it is fixed? Was it ever broken for them?

The issue with those was a different one, fixed in #6788

**Review by thomaseizinger — APPROVED** [review]

**Comment by jamilbk** [comment]
@dependabot rebase

**Pull request merged**